scripts/mkcloud

```diff
@@ -993,7 +1016,7 @@ function rebootcrowbar()
 {
     # reboot the crowbar instance
     #  and test if everything is up and running afterwards
-    sshrun "reboot"
+    sshrun "reboot >/dev/null &"
```

Comment:
I will fix the issue by adding `-oServerAliveInterval=2` to sshopts with the current PR I am working on.